main.go

```diff
@@ -1,12 +1,23 @@
 package main
 
-import "ldcli/cmd"
+import (
+	"net/http"
+	"time"
+
+	"ldcli/cmd"
+	"ldcli/internal/analytics"
+)
 
 // main.version is set at build time via ldflags by go releaser https://goreleaser.com/cookbooks/using-main.version/
 var (
 	version = "dev"
 )
 
 func main() {
-	cmd.Execute(version)
+	httpClient := &http.Client{
+		Timeout: time.Second * 3,
+	}
+	analyticsClient := &analytics.Client{HTTPClient: httpClient}
+	cmd.Execute(analyticsClient, version)
+	analyticsClient.Wait()
```

Comment:
This is created here so we can call `Wait()` to block the analytics request after everything else has finished.